DigitalLearningSolutions.Web/Views/SetPassword/Error.cshtml

```diff
@@ -0,0 +1,13 @@
+@{
+  ViewData["Title"] = "Set Password: Error";
+}
+
+@section NavMenuItems {
+  <partial name="_NavMenuItems" />
+}
+
+<h1 class="nhsuk-heading-xl">Something went wrong...</h1>
+<p class="nhsuk-body-m">
+  There was a problem resetting your password.
+  The set password link is only valid once, and expires after 2 hours.

```

Comment:
I added this to the ticket - can we make this link expire in three days instead of 2hours please (only for setting the pw, resetting should stay as it was)